src/core.cpp

```diff
@@ -371,7 +371,7 @@ bool Core::run_spinup()
         spunup = true;
         for( NameComponentIterator it = modelComponents.begin(); it != modelComponents.end(); ++it )
             spunup = spunup && ( *it ).second->run_spinup( step );
-        
+
```

Comment:
It's going to be a big drag if every PR now has tab/spacing wars with lots of changes like this. Ideally please make sure your RStudio settings match others'.

Reply:
I think this was due to RStudio removing leftover whitespace on save rather than the classic tab/space schism. I believe I fixed it in Rstudio's setting but I'm not sure how to undo the changes to the files.

Reply:
Personally, I would prefer to keep this version of the files -- it's good practice to trim trailing whitespace from our code. As far as project consistency, I think this is the whole point of the project-specific [`hector.Rproj`](https://github.com/JGCRI/hector/blob/master/hector.Rproj) file, which already specifies the removal of trailing whitespace, the enumber of spaces for indenting, etc.

I think the issue is that many of these C++ source files predate the Hector R package and therefore did not have these settings specified. @mnichol3 just happened to be the first person to open and re-save them since then.

For the contrarians among us who don't use RStudio (I think just me? Emacs!), perhaps we should also create a Hector [EditorConfig](https://editorconfig.org/) file to codify these settings?